src/Protocol/Models/InitializeParams.cs

```diff
@@ -57,7 +57,7 @@ public string RootPath
         /// The initial trace setting. If omitted trace is disabled ('off').
         /// </summary>
         [Optional]
-        public InitializeTrace Trace { get; set; } = InitializeTrace.Off;
+        public TraceValue TraceValue { get; set; } = TraceValue.Off;
```

Comment:
Will this rename break existing clients? Rename of property `Trace` to `TraceValue`.

Reply:
yes... that shouldn't be there, nice catch.  I renamed the enum to line up with the spec, but the property name should stay.